frontend/server/src/Controllers/Problem.php

```diff
@@ -5927,4 +6062,90 @@ public static function apiRandomProblem(\OmegaUp\Request $r) {
             'alias' => \OmegaUp\DAO\Problems::getRandomProblemAlias(),
         ];
     }
+
+    /**
+     * @param list<string> $tags
+     * @param list<string> $programmingLanguages
+     * @param array{0: int, 1: int}|null $difficultyRange
+     *
+     * @return array{column: string, columns: list<string>, currentTags: list<string>, keyword: string, language: string, languages: list<string>, mode: string, modes: list<string>, problems: list<ProblemListItem>, pagerItems: list<PagerItem>, tagData: list<Tag>, tags: list<string>}
```

Comment:
Tienes mucha razón, ya existe una anotación que contiene el tipo de `PageItem`, puedes usar esa anotación:
```suggestion
     * @return array{column: string, columns: list<string>, currentTags: list<string>, keyword: string, language: string, languages: list<string>, mode: string, modes: list<string>, problems: list<ProblemListItem>, pagerItems: list<PageItem>, tagData: list<Tag>, tags: list<string>}
```
y eliminar la que agregaste, ya sólo faltaría renombrar la del otro archivo